sql-plugin/src/main/scala/org/apache/spark/sql/rapids/datetimeExpressions.scala

```diff
@@ -1044,29 +1044,33 @@ class FromUTCTimestampExprMeta(
     rule: DataFromReplacementRule)
   extends BinaryExprMeta[FromUTCTimestamp](expr, conf, parent, rule) {
 
+  private[this] var timezoneId: ZoneId = null
+
+  lazy val supportedZoneIds = Seq("UTC", "America/Los_Angeles", "Asia/Shanghai")
+    .map(ZoneId.of(_).normalized)
+
   override def tagExprForGpu(): Unit = {
     extractStringLit(expr.right) match {
       case None =>
         willNotWorkOnGpu("timezone input must be a literal string")
       case Some(timezoneShortID) =>
         if (timezoneShortID != null) {
-          val utc = ZoneId.of("UTC").normalized
           // This is copied from Spark, to convert `(+|-)h:mm` into `(+|-)0h:mm`.
-          val timezone = ZoneId.of(timezoneShortID.replaceFirst("(\\+|\\-)(\\d):", "$10$2:"),
+          timezoneId = ZoneId.of(timezoneShortID.replaceFirst("(\\+|\\-)(\\d):", "$10$2:"),
             ZoneId.SHORT_IDS).normalized
 
-          if (timezone != utc) {
+          if (supportedZoneIds.forall(id => id != timezoneId)) {
             willNotWorkOnGpu("only timezones equivalent to UTC are supported")
```

Comment:
Does this message need to be changed to list all the supported timezones?

Reply:
Good catch. Updated